src/imageproxy.rs

```diff
@@ -318,6 +331,12 @@ pub struct ConvertedLayerInfo {
     pub media_type: oci_spec::image::MediaType,
 }
 
+/// Maps the two types of return values from the proxy
+enum FileDescriptors {
```

Comment:
The name of this is confusing.  It's more like "pipe and error-handle-thingy" or something, no?